### DISCUSSION TITLE
Link within reset password email [Jun 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2519] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### can u pls provide more details  
#  
> 💁 **o1lab**  
#### like what is the link you are getting and also about your setup.. like are u running it behind a ngnix proxy.. if so you would have to setup additional env variables  
#  
> 🙋‍♂️ **alex.pro**  
#### yes, I use nginx proxy and here is the link   
#### api/v1/db/auth/password/reset/db9b7475-37fc-4cb0-a253-23e8e9115aa9  
#  
> 🙋‍♂️ **alex.pro**  
#### <@310157751395287040>  which env vars should be set up?  
#  
> 🙋‍♂️ **alex.pro**  
#### can't find any info in google  
#  
> 💁 **o1lab**  
#### Nc_base_url  
#  
> 🙋‍♂️ **alex.pro**  
#### <@310157751395287040> Thanks, but I can't find any description here https://docs.nocodb.com/getting-started/installation/#environment-variables . Could you please help what should be put there?  
#  
> 🙋‍♂️ **alex.pro**  
#### - VIRTUAL_HOST   
#### - VIRTUAL_PORT   
#### - LETSENCRYPT_HOST   
#### - LETSENCRYPT_EMAIL   
#### - NC_DB=   
#### - NC_PUBLIC_URL=   
#### - NC_REQUEST_BODY_SIZE=  
#  
> 🙋‍♂️ **alex.pro**  
#### These env vars are set up  
#  
> 💁 **o1lab**  
#### Sorry, it is nc_public_url  
#  
> 💁 **o1lab**  
#### You can see the comment against it  
#  
> 💁 **o1lab**  
#### It mentions abt email  
#  
> 🙋‍♂️ **alex.pro**  
#### It started working without any update on my side  
#  
> 🙋‍♂️ **alex.pro**  
#### Magic  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/988737656642994186)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
